check_tv.py

```
@@ -452,12 +452,12 @@ def public_stream(live_id):  # Function to make stream public
        break  # Break loop
     except Exception as e:  # Handle exceptions
      if hitryagain == 3:  # Check if retry limit is reached
-      logging.info(f"Error and stoping because of error that can't fix")  # Log error
+      logger.info(f"Error and stoping because of error that can't fix")  # Log error
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)